packages/hash/frontend/src/pages/shared/ory-kratos.ts

```diff
@@ -1,37 +1,24 @@
 import {
   Configuration,
-  V0alpha2Api,
-  SelfServiceError,
-  SelfServiceLoginFlow,
-  SelfServiceRegistrationFlow,
-  SelfServiceBrowserLocationChangeRequiredError,
   ErrorAuthenticatorAssuranceLevelNotSatisfied,
+  ErrorBrowserLocationChangeRequired,
+  FrontendApi,
   NeedsPrivilegedSessionError,
+  LoginFlow,
+  RecoveryFlow,
+  RegistrationFlow,
+  SettingsFlow,
+  VerificationFlow,
+  UiNodeInputAttributes,
 } from "@ory/client";
 import { isUiNodeInputAttributes } from "@ory/integrations/ui";
 import { AxiosError } from "axios";
 import { NextRouter } from "next/router";
 import { Dispatch, SetStateAction } from "react";
+import { edgeConfig } from "@ory/integrations/next";
 
-type SelfServiceFlow = SelfServiceLoginFlow | SelfServiceRegistrationFlow;
-
-export const oryKratosClient = new V0alpha2Api(
-  new Configuration({
-    /**
-     * Directly connecting to kratos (using "http://127.0.0.1:4433") would prevent the
-     * CRSF token from being set as an HTTP-Cookie, because the browser cannot send or
-     * receive cookies via the browser `fetch` method unless:
-     *  1. `credentials: "include"` is set in the HTTP request header
-     *  2. the correct CORS origin is configured in the kratos server
-     *
-     * Therefore requests to the ory kratos public endpoint are made on the server in a
-     * Next.js API handler.
-     */
-    basePath: "/api/ory",
```

Comment:
This is going to conflict with changes made in [this PR](https://github.com/hashintel/hash/pull/1637) to make the `oryKratosClient` work on the server-side. Do you know whether this will work on the server-side?

Reply:
It would not, no. The change here isn't required, so I'll undo it. It just defined the same config we had already. I'll rename the endpoint again to `/api/ory` from `/api/.ory`

Reply:
Reverted in bbde5d84ba5f6797e3e4c5e64d53f4c68f1ec306